web-console/src/utils/druid-query.ts

```diff
@@ -92,12 +97,55 @@ export class DruidError extends Error {
     return;
   }
 
+  static getSuggestion(errorMessage: string): QuerySuggestion | undefined {
+    // == is used instead of =
+    if (errorMessage.includes('Encountered "= =" at')) {
+      return {
+        label: `Replace == with =`,
+        fn: str => {
+          if (!str.includes('==')) return;
+          return str.replace(/==/g, '=');
```

Comment:
what if `==` is part of a literal? Does `str#replace` replace all instances of the regex in the string?

Reply:
yeah it is pretty blunt, just replaces all the `==` with `=`... Maybe it should be aware of the line markers in the error text